src/Pages/InterestPage/trackerItem.js

```diff
@@ -0,0 +1,22 @@
+import uncheck from '../../assets/images/uncheck.png';
+import bin from '../../assets/images/delete.png';
+
+function TrackerItem({ item, changeItem, deleteItem, date }) {
```

Comment:
why are u getting this props "changeItem, deleteItem" here?

Reply:
it's not needed right now just in case i need it later if not i will remove them

Reply:
instead of "item" call the prop "title"